documentation/docs/decentralized_identifiers/update.mdx

```diff
@@ -12,19 +12,382 @@ keywords:
 ---
 import account_manipulate_did_rs from '!!raw-loader!../../../examples/account/manipulate_did.rs';
 import CodeSnippet from '../../src/components/CodeSnippetComponent'
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import CodeBlock from '@theme/CodeBlock';
 
 
-TODO: Explain how DID Documents can be manipulated and how updates should be published.
-
 ## Example
 
 This example shows how you can add more to an existing DID Document. The two main things you can add are Verification Methods and Services. A verification method adds public keys, which can be used to digitally sign things like an identity. The services provide metadata around the identity via URIs. These can be URLs, but also emails or IOTA indices.
 
-### Account Module (Recommended)
-
 <CodeSnippet 
     nodeReplitLink="https://repl.it/@IOTAFoundation/manipulate-did?lite=true"
     rustContent={account_manipulate_did_rs}
     nodeGithubLink = "https://github.com/iotaledger/identity.rs/blob/dev/bindings/wasm/examples-account/src/manipulate_did.ts"
     rustGithubLink = "https://github.com/iotaledger/identity.rs/blob/dev/examples/account/manipulate_did.rs"
 />
+
+### 1. Creating Identity
+
+The Example above starts by creating an identity using the account.
+See [create identity example](./create.mdx).
+
+<Tabs>
+<TabItem value="Rust" label="Rust">
+
+```rust
+let mut account: Account = Account::builder()
+    .storage(stronghold)
+    .create_identity(IdentitySetup::default())
+    .await?;
+```
+
+</TabItem>
+<TabItem value="node" label="Node.js">
+
+```js
+let builder = new AccountBuilder({
+    storage,
+});
+let account = await builder.createIdentity();
+```
+</TabItem>
+</Tabs>
+
+This will create a DID document and publish it to the tangle.
+
+```json
+{
+   "doc":{
+      "id":"did:iota:6T4PRHWp7bNsKaBWr1gVtUBQfLaxAKqKGAeJWFBZkMyf",
+      "capabilityInvocation":[
+         {
+            "id":"did:iota:6T4PRHWp7bNsKaBWr1gVtUBQfLaxAKqKGAeJWFBZkMyf#sign-0",
+            "controller":"did:iota:6T4PRHWp7bNsKaBWr1gVtUBQfLaxAKqKGAeJWFBZkMyf",
+            "type":"Ed25519VerificationKey2018",
+            "publicKeyMultibase":"z5k7vzMVuXXj8MJDcfzP2owvc8xKBA6BBsAkFf1GSNu2X"
+         }
+      ]
+   },
+   "meta":{
+      "created":"2022-04-13T09:27:48Z",
+      "updated":"2022-04-13T09:27:48Z"
+   }
+}
+```
+The created document only contains one verification method with [capabilityInvocation](https://www.w3.org/TR/did-core/#capability-invocation) relationship.
+This method is used to sign the document, effectively controlling the DID.
+
+Any future updates to the document in this example will be signed using this verification method. The Account takes care of this automatically so we don't need to think about signing each update.
```

Comment:
Perhaps we should add that it is possible to rotate the key:

"...signing each update. It is possible to rotate a `capabilityInvocation` key, in which case the Account will correctly sign the next update with a key which was valid in the previous state of the DID Document. Afterwards it will start using the new key automatically. If multiple `capabilityInvocation` keys are managed by an Account, it will sign with the first it has control over" (We should verify how that behavior works before we mention it)